### ISSUE TITLE
2021-6-16

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#577]
220理解与理论：人工智能基础问题的悲观与乐观 https://mp.weixin.qq.com/s/viCB0BcVy8fXMGK4Kn0IlQ (2021-6-11)
221How to compute start and end time of python program? time.time() - start_time; (2021-6-11)
222The strange thing is, the loss does not drop; reset learning rate; SOLVED; separate learning_rate; done; (2021-6-11)
223Why the frequency of corpus in python is not the same as we used in website? Because the corpus we used is just part of whole database; (2021-6-11)
224Change name word to combined_t_sentence; done; delete the strange UNK; done; delete counts_dict; give up, because we need frequency; (2021-6-12)
225鲁白：AI防疫专题论坛开幕致辞 https://hub.baai.ac.cn/view/1652 (2021-6-12)
226GNN做random walk; (2021-6-12)
227https://cs.stanford.edu/~jure/ (2021-6-12)
228郭毅可：抗疫政策的评估：数据科学的应用 https://hub.baai.ac.cn/view/1653 预测武汉疫情; 传播动态系统数学模型; (2021-6-13)
229What is 预训练? 预训练通过自监督学习从大规模数据中获得与具体任务无关的预训练模型。体现某一个词在一个特定上下文中的语义表征。第二个步骤是微调，针对具体的任务修正网络。// 首先，预训练模型是一种迁移学习的应用，利用几乎无限的文本，学习输入句子的每一个成员的上下文相关的表示，它隐式地学习到了通用的语法语义知识。第二，它可以将从开放领域学到的知识迁移到下游任务，以改善低资源任务，对低资源语言处理也非常有利。第三，预训练模型在几乎所有 NLP 任务中都取得了目前最佳的成果。最后，这个预训练模型+微调机制具备很好的可扩展性，在支持一个新任务时，只需要利用该任务的标注数据进行微调即可，一般工程师就可以实现。(2021-6-12)
